.gitignore

```diff
@@ -118,3 +118,5 @@ ENV/
 app.sqlite
 app/local_settings.py
 
+#Uploaded files
+uploads
```

Comment:
Please change this to uploads/ to include the entire directory

Reply:
Also, if we ever intend the end user to request the image, it needs to be under static/